docs/docsite/rst/dev_guide/testing/sanity/type-boilerplate.rst

```diff
@@ -0,0 +1,8 @@
+Sanity Tests » type boilerplate
+===============================
+
+Most Python files should include the following boilerplate:
```

Comment:
It would be helpful to mention that this must come after any `__future__` import, but before any class definitions.

I'd also briefly explain why it's important.

Reply:
type-boilerplate now starts with:

> Most Python files should include the following boilerplate at the top of the file, right after the
comment header and ``from __future__ import``:

and an explanation of what that does:
https://github.com/ansible/ansible/pull/58641/files#diff-e27c8e1cc94b9db801f598584e2d4a58